web/org.eclipse.xtext.web/src/main/java/org/eclipse/xtext/web/server/persistence/FileResourceHandler.xtend

```diff
@@ -10,8 +10,10 @@ package org.eclipse.xtext.web.server.persistence
 import com.google.inject.Provider
 import java.io.IOException
 import java.io.OutputStreamWriter
+import java.util.List
```

Comment:
I think this file change was probably accidentally committed as part of this PR? It's just whitespace and additional (un-used) import, and has nothing to do with this PR (Xtend Web Editor) - perhaps you would like to undo to remove this changed file from this PR?